frontend/integration-tests/tests/kubevirt/kubevirt.login.scenario.ts

```diff
@@ -0,0 +1,19 @@
+import { browser } from 'protractor';
+import { appHost } from '../../protractor.conf';
+import { execSync } from 'child_process';
+import { KUBEADMIN_IDP, KUBEADMIN_USERNAME, BRIDGE_KUBEADMIN_PASSWORD } from './utils/consts';
+import * as loginView from '../../views/login.view';
+
+
+describe('Authentication', () => {
+  beforeAll(async() => {
+    await browser.get(appHost);
+    await browser.sleep(3000); // Wait long enough for the login redirect to complete
+  });
+
+  it('Web console logs in.', async() => {
```

Comment:
I'm still unclear on why we need a separate login scenario just for kubevirt. I'd rather not fork the login test code unless there is a very compelling reason.

Reply:
It just has one more line about `oc login` as our current tests requires.

Reply:
You don’t need to use oc login for that. You just need the KUBECONFIG env var set. Admin console tests also run cli commands

Reply:
Then we don't need another login. Could you comment on #2016?